ilastik/applets/objectClassification/objectClassificationDataExportGui.py

```diff
@@ -113,6 +114,10 @@ def setupLayers(self):
                 layer.visible = False
                 layer.name = layer.name + "- Preview"
             layers += previewLayers
+        elif selection == "Object Identities":
+            previewSlot = self.topLevelOperatorView.ImageToExport
+            layer = self._initColortableLayer(previewSlot)
```

Comment:
Seems like it'd make more semantic sense if `_initColortableLayer` would return a single layer (as the singular name implies), and then just add it to the `layers`  list with `.append` or whatever. Or perhaps you're expecting `_initColortableLayer` to eventually return more than one layer? In that case, maybe that should be pluralized?